components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -201,19 +192,16 @@ func (k *KnativeLib) UpdateSubscription(sub *evapisv1alpha1.Subscription) (*evap
 
 // SendMessage sends a message to a channel
 func (k *KnativeLib) SendMessage(channel *evapisv1alpha1.Channel, headers *map[string][]string, payload *string) error {
-
+	httpClient := &http.Client{
+		Transport: initHTTPTransport(),
+	}
```

Comment:
What was the motivation behind the replacement of that singleton?

Reply:
commit b59a48a56125502abd0a71ddae99008d10e65164 was missing on this PR, this issue has been addressed by Rafaey.